trio/_core/_run.py

```diff
@@ -1705,13 +1701,17 @@ def spawn_impl(
             assert self.init_task is None
 
         ######
-        # Propagate contextvars
+        # Propagate contextvars, and make sure that async_fn can use sniffio.
```

Comment:
Err, maybe a bad merge happened or something?

Reply:
Yes, very bad, but it's fixed now.